Sources/UIApplication+handleSDLEvents.swift

```diff
@@ -101,25 +101,35 @@ extension UIApplication {
         #endif
 
         UIApplication.shared?.delegate?.applicationWillEnterForeground(UIApplication.shared)
+        UIApplication.post(willEnterForegroundNotification)
     }
 
     static func onDidEnterForeground() {
         UIApplication.shared?.delegate?.applicationDidBecomeActive(UIApplication.shared)
+        UIApplication.post(didBecomeActiveNotification)
     }
 
     static func onWillEnterBackground() {
         UIApplication.shared?.delegate?.applicationWillResignActive(UIApplication.shared)
+        UIApplication.post(willResignActiveNotification)
     }
 
     static func onDidEnterBackground() {
         UIApplication.shared?.delegate?.applicationDidEnterBackground(UIApplication.shared)
+        UIApplication.post(didEnterBackgroundNotification)
 
         #if os(Android)
         UIScreen.main = nil
         #endif
     }
 }
 
+private extension UIApplication {
+    static func post(_ name: NSNotification.Name) {
```

Comment:
does it make sense to make these `class func`s too? (also what is currently `static func onDidEnterBackground() {` etc?)

I think we won't win much from making `willEnterForegroundNotification` into `class var`s otherwise